docs/commands.rst

```diff
@@ -608,49 +613,66 @@ Here is an example command to covert a job run XML into JUnit results.
 
  xsltproc job2junit.xml simple_job.07/job.xml > simple_job.07/junit.xml
 
+.. _legacy_rhts_cmds:
+
 Legacy RHTS Commands
 --------------------
 
-If you have the restraint-rhts subpackage installed these commands are provided
-in order to support legacy tests written for RHTS.
-
-rhts-backup
-~~~~~~~~~~~
-
-Use `rstrnt-backup` instead.
-
-rhts-environment.sh
-~~~~~~~~~~~~~~~~~~~
+The table below lists known legacy rhts commands.  Some are provided in the 
+restraint-rhts package and some are from rhts packages.  It is encouraged for people
+to use restraint's substitute for these commands as they are actively supported.
+Included in the table are the restraint substitutes and which rhts commands are deprecated.
+
++--------------------------------+-------------------------------------------+------------+
+| RHTS Legacy Script             | Restraint Substitute                      | Deprecated |
```

Comment:
Not sure about the Deprecated. All rhts commands should be deprecated, shouldn't they?

Reply:
There is a substitute for RHTS or no substitute which is where 'deprecated' applies.

Reply:
Deprecated column has been removed.